sympy/solvers/solvers.py

```diff
@@ -1136,6 +1137,26 @@ def _has_piecewise(e):
         if _has_piecewise(fi):
             f[i] = piecewise_fold(fi)
 
+    # expand double angles; in general, expand_trig will allow
+    # more roots to be found but this is not a great solultion
+    # to not returning a parametric solution, otherwise
+    # many values can be returned that have a simple
+    # relationship between values
+    targs = {t for fi in f for t in fi.atoms(TrigonometricFunction)}
+    add, other = sift(targs, lambda x: x.args[0].is_Add, binary=True)
+    trep = {}
+    for i in add:
+        a = i.args[0]
+        if len(a.args) == 2:
+            ind, dep = a.as_independent(*symbols)
+            if dep in symbols or -dep in symbols:
```

Comment:
```suggestion
            if dep in symbols or -dep in symbols and ind.is_number:
```

Reply:
Maybe only the sin/cos addition formulae should be used (`TR10`?)